src/main/java/lottoauto/domain/lotto/CustomShuffle.java

```diff
@@ -0,0 +1,9 @@
+package lottoauto.domain.lotto;
+
+import java.util.List;
+
+@FunctionalInterface
+public interface CustomShuffle {
+
+    List<Integer> makeShuffle();
```

Comment:
LottoNumber 객체를 활용하면 어떨까요?
```suggestion
    List<LottoNumber> makeShuffle();
```

Reply:
사실 이 부분을 수정하면서 Integer 로 할지 LottoNumber 로 할지 정말 많이 고민했습니다. 

처음 생각은 CustomShuffle 인터페이스는 순수하게 숫자만 shuffle 해주는 역할을 하게 하기 위해 Integer로 리턴하고 받은 숫자를 Lotto 클래스에서 Lotto 클래스를 생성할 때 번호가 6개이고 중복이 없는지를 확인하려고 했었는데 아래 피드백 내용중에 

```
src/main/java/lottoauto/domain/lotto/LottoShuffler.java

private final List<Integer> defaultLottoNumbers =  IntStream.range(1, 46)
            .boxed()
            .collect(Collectors.toList());
```

이 부분을 먼저 LottoNumber로 변환해서 내부 필드로 저장하고 있으면 LottoShuffler 클래스에 대한 단위 테스트도 훨씬 작성하기 편해지고 Lotto 클래스의 생성자에서 중복이나 6개의 숫자가 맞는지에 대한 코드도 줄어들 수 있을것 같아서 말씀해주신 것처럼 바꾸는게 훨씬 좋은것 같습니다. 

다음 미션에는 해당 내용 반영해서 진행하도록 하겠습니다.
감사합니다